<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="40092" started-at="2022-03-21T10:30:04Z" finished-at="2022-03-21T10:30:44Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="40092" started-at="2022-03-21T10:30:04Z" finished-at="2022-03-21T10:30:44Z">
      <class name="tech.com.FlipKartPro">
        <test-method status="PASS" signature="startBrowser()[pri:0, instance:tech.com.FlipKartPro@4667ae56]" name="startBrowser" is-config="true" duration-ms="14923" started-at="2022-03-21T10:30:04Z" finished-at="2022-03-21T10:30:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startBrowser -->
        <test-method status="FAIL" signature="Project()[pri:0, instance:tech.com.FlipKartPro@4667ae56]" name="Project" duration-ms="24076" started-at="2022-03-21T10:30:19Z" finished-at="2022-03-21T10:30:43Z">
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index: 1, Size: 1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
at java.util.ArrayList.rangeCheck(Unknown Source)
at java.util.ArrayList.get(Unknown Source)
at tech.com.FlipKartPro.addToCart(FlipKartPro.java:64)
at tech.com.FlipKartPro.Project(FlipKartPro.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Project -->
        <test-method status="PASS" signature="close()[pri:0, instance:tech.com.FlipKartPro@4667ae56]" name="close" is-config="true" duration-ms="1019" started-at="2022-03-21T10:30:43Z" finished-at="2022-03-21T10:30:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- tech.com.FlipKartPro -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
